README.md

```diff
@@ -198,6 +198,9 @@ You can also load your JSONified config in from an ENV variable (e.g. `KUBE_CONF
 ```
 Kubeclient::Config.new(JSON.parse(ENV['KUBE_CONFIG']), nil)
 ```
+###Supported kubernetes versions
+
+For 1.1 only the core api is supported, apis/extensions are supported in later versions.
```

Comment:
added 'v1'. @cben the point is that we know that for OSE 3.1 ( - kubernetes v1.1.0-origin ) kind is missing, so anyway we can't guarantee support for 1.1 and that's what I have mentioned in the readme.